src/Mod/Spreadsheet/App/Cell.cpp

```diff
@@ -41,6 +41,7 @@
 #include <App/ExpressionParser.h>
 #include "Sheet.h"
 #include <iomanip>
+#include <cctype>
```

Comment:
I think you also need to include <memory>, to get `make_unique` (Travis is complaining about it in the Linux builds -- it's probably in the PCH on Windows).
```suggestion
#include <cctype>
#include <memory>
```

Reply:
I will try to solve this with `#include <Base/StdStlTools.h>` instead, see comment above.